css/style.css

```diff
@@ -0,0 +1,115 @@
+/*style.css*/
+* {
+     -moz-box-sizing: border-box; /* Firexfox */
+     -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
+     box-sizing: border-box; /* IE */
+}
+
+.clearfix:after {
+     content: ".";
+     clear: both;
+}
+
+.clear {
+  clear: both;
+}
+
+.alignright {
+  float: right;
+  padding: 0 0 10px 10px;
+}
+
+.alignleft {
+  float: left;
+  padding: 0 10px 10px 0;
+}
+
+.wrapper {
+  width: 100%;
+  margin: 0px auto;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 300;
+  font-size: 12px;
+  border: 5px solid white;
+}
+
+.header {
+  background-color: #F0F8FF;
+  font-size: 20px;
+    position: relative;
+}
+
+.header img {
+  padding: 30px 500px 10px;
+}
+
+
+.nav li { /* better way to make this more responsive & keep it centered?*/
+  display: inline-block;
+  padding: 10px;
+}
+
+.about {
+  padding-left: 500px;
+}
+
+.menu {
+  padding-right: 500px;
```

Comment:
It seems with these like you're trying to achieve horizontal centering. Just make the parent container display block and use `text-align: center` to center the child content.